plugin/src/leiningen/cljs_lambda/aws.clj

```diff
@@ -39,9 +44,9 @@
                               "]"])}))))
 
 (defn ->cli-args [m & [positional {:keys [preserve-names?]}]]
-  (let [m    (cond-> (merge (meta-config) (update-vpc-value m))
+  (let [m    (cond-> (merge (meta-config) (-> m (update-vpc-value) (update-dead-letter-value)))
                (not preserve-names?)
-               (set/rename-keys {:name :function-name :vpc :vpc-config}))
```

Comment:
I meant, feel free to merge this one and I'll do mentioned changes along with env in another pull request.